lib/insights.ts

```diff
@@ -65,6 +66,8 @@ class AlgoliaAnalytics {
   // LocalStorage
   storageManager: StorageManager;
 
+  version: string = version;
```

Comment:
should it be VERSION instead?

Reply:
How do you read this? I think we have mixed between them here and there

Reply:
`version` is fine to me.

Reply:
> How do you read this? 
You can read this in the console with AlgoliaInsights.version. 
I added it because instantsearch has the same. I don't really need it for the user agent.